tide_core.module

```diff
@@ -288,6 +303,41 @@ function tide_core_editor_js_settings_alter(array &$settings) {
 function tide_core_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   $form['#attached']['library'][] = 'tide_core/sticky_node_form_sidebar';
   $form['#process'][] = '_tide_core_form_node_form_process';
+
+  // Add comment log message field.
+  $node = $form_state->getFormObject()->getEntity();
+  // Apply to edit form only, look tide_workflow_notification_form_node_form_alter.
+  if ($node->isNew()) {
+    return;
+  }
+  if (isset($form['revision_log']) && $form['revision_log']['#access']) {
+    $form['moderation_state']['comment_log_message'] = [
+      '#type' => 'textarea',
+      '#title' => t('Change request detail:'),
+      '#description' => t('Specify details of <strong>all changes</strong> made in this update. If content is embargoed or time specific, enter the scheduling details. Content will be published within <strong>24 business hours</strong>. Contact your content team for urgent publishing requests.'),
+      '#rows' => 3,
+      '#weight' => 9,
+    ];
+    $form['#validate'][] = '_tide_core_node_edit_form_log_message_validate';
+    $form['#attached']['library'][] = 'tide_core/node_revision_log';
+  }
+
+}
+
+/**
+ * Node edit form validate function.
```

Comment:
This also needs a proper docblock comment IMO -- surprised it wasn't picked up by PHPCS